src/main/resources/application.yml

```diff
@@ -198,6 +198,8 @@ FAMILIE_EF_PROXY_URL: https://familie-ef-proxy.${ON_PREM_URL_ENV}-fss-pub.nais.i
 
 FAMILIE_KS_SAK_URL: http://familie-ks-sak
 
+FAMILIE_BA_SAK_URL: http://familie-ba-sak
```

Comment:
Trenger oppsett for token-support også under 

```
no.nav.security.jwt:
  client:
    registration:
    ...
```

Reply:
Bra spotta! Fisa i [Legg til oppsett for token-support](https://github.com/navikt/familie-klage/pull/236/commits/7cf0dfaae92a5b7bda3297fa80b103f4c9d2b76d)